### PR TITLE
W-13796916 loading error (browser extension)

### PR DESCRIPTION
**Pull request by dukesphere**

### CODE REVIEW

File: modules/ROOT/pages/app-loading-errors.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
+= Application Loading Errors
+
+Errors sometimes occur when you open an application in Anypoint Code Builder. 
+
+== Loading Error in the UI Canvas
```
Comment (pablocabrera85) [inline modules/ROOT/pages/app-loading-errors.adoc]: This does not only apply to the UI canvas, but also to the create project UIs

File: modules/ROOT/pages/app-loading-errors.adoc [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+A *Loading...* error in the UI canvas of an open application typically indicates that a browser extension is interfering with the UI loading process. 
+
+To address this issue, try:
+
+* Disabling browser extensions and reloading the application 
+* Using another browser that is not using extensions
+
+
+== Flow Canvas Preparation Never Completes
```
Comment (pablocabrera85) [inline modules/ROOT/pages/app-loading-errors.adoc]: When the users open a different folder, the UI will finish loading but show the same content as when the XML has no flows. We should rephrase the title to something on the line of. "UI rendered as if my app has no flows"

File: modules/ROOT/pages/troubleshoot-app-loading-errors.adoc [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+* Using another browser that is not using extensions
+
+
+== Flow Canvas Preparation Never Completes
+
+If the UI canvas contains the message *Preparing Flow Canvas* but the flow graphics fail to load after a few minutes, make sure that your application is open from its _root_ directory or from a multi-root workspace. 
+
+Visual Studio Code treats a project directory as a workspace when you open it to the project's root directory or if your project folder is located in a https://code.visualstudio.com/docs/editor/multi-root-workspaces[multi-root workspace^].
+
+. Open the Command Palette by pressing Ctrl+Shift+p (Windows) or Cmd+Shift+p (Mac), and provide the following command:
```
Comment (dmerlob) [inline modules/ROOT/pages/troubleshoot-app-loading-errors.adoc:30]:
```suggestion
. Open the Command Palette by pressing `Ctrl+Shift+p` (Windows) or `Cmd+Shift+p` (Mac), and provide the following command:
```

**Review by dmerlob — APPROVED** [review]

File: modules/ROOT/pages/troubleshoot-app-loading-errors.adoc (hunk 3 above)
Comment (dukesphere) [reply modules/ROOT/pages/troubleshoot-app-loading-errors.adoc:30]: @dmerlob Apparently, the Salesforce style does not use a different font face for the key command shortcuts. I don't know why, and it's unintuitive to me, but it is what it is. ¯\_(ツ)_/¯ Hanna pointed me to that guidance.

**Comment by dukesphere** [comment]
Pablo's comments are incorporated.

File: modules/ROOT/pages/troubleshoot-app-loading-errors.adoc [hunk 4]
```diff
@@ -0,0 +1,41 @@
+= Application Loading Errors
+
+include::reuse::partial$beta-banner.adoc[tag="anypoint-code-builder"]
+
+//LOGO (web, desktop, or both)
+include::partial$acb-ide-logos.adoc[tags="both-ides"]
+
+Errors sometimes occur when you open an application in Anypoint Code Builder. 
+
+== Loading Error in the UI Canvas
```
Comment (dukesphere) [inline modules/ROOT/pages/troubleshoot-app-loading-errors.adoc]: @pablocabrera85 Can you clarify your comment here:

> This does not only apply to the UI canvas, but also to the create project UIs

Comment (dukesphere) [reply modules/ROOT/pages/troubleshoot-app-loading-errors.adoc]: Does the Loading... error occur, or what happens? 

I'm not clear what the difference between "project UIs" and "UI canvas" is. Do you mean some other parts of the open project might not load? Can you show me the error?

File: modules/ROOT/pages/troubleshoot-app-loading-errors.adoc [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+A *Loading...* error in the UI canvas of an open application typically indicates that a browser extension is interfering with the UI loading process. 
+
+To address this issue, try:
+
+* Disabling browser extensions and reloading the application 
+* Using another browser that is not using extensions
+
+
+== Flow Canvas Preparation Never Completes
```
Comment (dukesphere) [inline modules/ROOT/pages/troubleshoot-app-loading-errors.adoc]:
```suggestion
== Canvas UI Renders Without Configured Flows
```

File: modules/ROOT/pages/troubleshoot-app-loading-errors.adoc [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+If the UI canvas contains the message *Preparing Flow Canvas* but the flow graphics fail to load after a few minutes, make sure that your application is open from its _root_ directory or from a multi-root workspace.
```
Comment (dukesphere) [inline modules/ROOT/pages/troubleshoot-app-loading-errors.adoc]:
```suggestion
If the UI canvas for an application that is already configured with a flow or subflow treats your application as a new project, prompting you with the question *What do you want to build in this file?* and *Flow* and *Sub flow* creation options, make sure that your application is open from its _root_ directory or from a multi-root workspace. 
```

File: modules/ROOT/pages/troubleshoot-app-loading-errors.adoc [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+A *Loading...* error in the UI canvas of an open application typically indicates that a browser extension is interfering with the UI loading process. 
+
+To address this issue, try:
+
+* Disabling browser extensions and reloading the application 
+* Using another browser that is not using extensions
+
+
+== Canvas UI Renders Without Configured Flows
```
Comment (dukesphere) [inline modules/ROOT/pages/troubleshoot-app-loading-errors.adoc]:
```suggestion
== Canvas UI Renders Without Configured Flows or Subflows
```

File: modules/ROOT/pages/troubleshoot-app-loading-errors.adoc [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+include::reuse::partial$beta-banner.adoc[tag="anypoint-code-builder"]
+
+//LOGO (web, desktop, or both)
+include::partial$acb-ide-logos.adoc[tags="both-ides"]
+
+Errors sometimes occur when you open an application in Anypoint Code Builder. 
+
+== Loading Error in the UI Canvas
+
+A *Loading...* error in the UI canvas of an open application typically indicates that a browser extension is interfering with the UI loading process.
```
Comment (dukesphere) [inline modules/ROOT/pages/troubleshoot-app-loading-errors.adoc]:
```suggestion
A *Loading...* error in the UI of an open application typically indicates that a browser extension is interfering with the UI loading process. 
```

File: modules/ROOT/pages/troubleshoot-app-loading-errors.adoc (hunk 4 above)
Comment (dukesphere) [reply modules/ROOT/pages/troubleshoot-app-loading-errors.adoc]: Changed to "Loading Error in the Application UI"

File: modules/ROOT/pages/app-loading-errors.adoc (hunk 2 above)
Comment (dukesphere) [reply modules/ROOT/pages/app-loading-errors.adoc]: changed to "== Canvas UI Renders Without Configured Flows or Subflows"

**Comment by dukesphere** [comment]
review comments from Pablo are incorporated

File: modules/ROOT/pages/troubleshoot-app-loading-errors.adoc [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+* Using another browser that is not using extensions
+
+
+== Canvas UI Renders Without Configured Flows or Subflows
+
+If the UI for an application that is already configured with a flow or subflow treats your application as a new project, prompting you with the question *What do you want to build in this file?* and *Flow* and *Sub flow* creation options, make sure that your application is open from its _root_ directory or from a multi-root workspace. 
+
+image::acb-start-flow-subflow-ui.png["Start Screen Building a Mule Application"]
+
+Visual Studio Code treats a project directory as a workspace when you open it to the project's root directory or if your project folder is located in a https://code.visualstudio.com/docs/editor/multi-root-workspaces[multi-root workspace^].
```
Comment (pablocabrera85) [inline modules/ROOT/pages/troubleshoot-app-loading-errors.adoc]: I find this a bit hard to follow. What do you think of something around `Visual Studio Code treats an open directory as a workspace. Even if this folder contains several projects, they won't be treated as such. In order to have multiple projects open in the same editor, you need to create a https://code.visualstudio.com/docs/editor/multi-root-workspaces[multi-root workspace^].`

Comment (dukesphere) [reply modules/ROOT/pages/troubleshoot-app-loading-errors.adoc]: ok, I think I see what you mean. thx

Comment (dukesphere) [reply modules/ROOT/pages/troubleshoot-app-loading-errors.adoc]:
```suggestion
Visual Studio Code treats an open directory as a workspace. For the project to load into a workspace, the root directory of the project must be the top-level directory in the Explorer view, or the project must belong to a https://code.visualstudio.com/docs/editor/multi-root-workspaces[multi-root workspace^].
```

File: modules/ROOT/pages/troubleshoot-app-loading-errors.adoc [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+* Using another browser that is not using extensions
+
+
+== Canvas UI Renders Without Configured Flows or Subflows
+
+If the UI for an application that is already configured with a flow or subflow treats your application as a new project, prompting you with the question *What do you want to build in this file?* and *Flow* and *Sub flow* creation options, make sure that your application is open from its _root_ directory or from a multi-root workspace. 
+
+image::acb-start-flow-subflow-ui.png["Start Screen Building a Mule Application"]
+
+Visual Studio Code treats an open directory as a workspace. For the project to load into a workspace, the root directory of the project must be the top-level directory in the Explorer view, or the project must belong to a https://code.visualstudio.com/docs/editor/multi-root-workspaces[multi-root workspace^].
```
Comment (dukesphere) [inline modules/ROOT/pages/troubleshoot-app-loading-errors.adoc:28]: revised @pablocabrera85 
can you take a look

**Review by pablocabrera85 — APPROVED** [review]
